docs/content/querying/lookups.md

```diff
@@ -296,3 +296,157 @@ To test this setup, you can send key/value pairs to a kafka stream via the follo
 ```
 
 Renames can then be published as `OLD_VAL->NEW_VAL` followed by newline (enter or return)
+
+Dynamic configuration
+---------------------
+
+The following documents the behavior of the cluster-wide config which is accessible through the coordinator.
+The configuration is propogated through the concept of "tier" of servers.
+A "tier" is defined as a group of services which should receive a set of lookups.
+For example, you might have all historicals be part of `__default`, and Peons be part of individual tiers for the datasources they are tasked with.
+
+These configs are accessed using JSON through the following URI template
+
+```
+http://<COORDINATOR_IP>:<PORT>/druid/coordinator/v1/lookups/{tier}/{id}
+```
+
+All URIs below are assumed to have `http://<COORDINATOR_IP>:<PORT>` prepended.
+
+If you have NEVER configured lookups before, you MUST post an empty json object `{}` to `/druid/coordinator/v1/lookups` to initialize the configuration.
```

Comment:
but this is not intuitive for user, can we make sure that when the coordinator starts it does what it needs to make it working, instead of relying on users. Also what happen if this query fails ? how do you deal with that ?